### ISSUE TITLE
Can't connect - Unrecognized option 'crtscts'

### ISSUE THREAD

**Issue by Justintime50** [mobilejazz/docker-ipsec-vpn-server#20]
I can't get this working out of the box. I've followed the readme guide, I've looked into the troubleshooting steps, and nothing seems to help. The only thing I can think is something fails during connection as shown here in the logs. The container starts up in Docker like it should but doesn't accept connections. I've created a user, reset the container, reset the client machine, deleted and re-added the user, and exposed the proper ports with no luck. 

I'm trying to connect from a Mac, I've also tried connecting from an iPhone and get the same error message: 
```
You were disconnected by the communication device. Try reconnecting. If the problem continues, verify your settings.
```

**LOGS**
```
2019-06-23T05:54:33.967495480Z 
Trying to auto discover IPs of this server...
Warning: ignored obsolete keyword 'nat_traversal'
Warning: ignored obsolete keyword 'nat_traversal'
Warning: obsolete keyword 'modecfgdns2' ignored
Redirecting to: /etc/init.d/ipsec start
Warning: ignored obsolete keyword 'nat_traversal'
Warning: ignored obsolete keyword 'nat_traversal'
Warning: obsolete keyword 'modecfgdns2' ignored
Starting pluto IKE daemon for IPsec: Initializing NSS database
2019-06-23T05:54:35.265489308Z 
Warning: ignored obsolete keyword 'nat_traversal'
.
xl2tpd[1]: setsockopt recvref[30]: Protocol not available
xl2tpd[1]: Using l2tp kernel support.
xl2tpd[1]: xl2tpd version xl2tpd-1.3.8 started on 4c6e2d49c6b7 PID:1
xl2tpd[1]: Written by Mark Spencer, Copyright (C) 1998, Adtran, Inc.
xl2tpd[1]: Forked by Scott Balmos and David Stipp, (C) 2001
xl2tpd[1]: Inherited by Jeff McAdams, (C) 2002
xl2tpd[1]: Forked again by Xelerance (www.xelerance.com) (C) 2006-2016
xl2tpd[1]: Listening on IP address 0.0.0.0, port 1701
xl2tpd[1]: Connection established to 172.17.0.1, 49893.  Local: 47807, Remote: 7 (ref=0/0).  LNS session is 'default'
xl2tpd[1]: start_pppd: I'm running: 
xl2tpd[1]: "/usr/sbin/pppd" 
xl2tpd[1]: "plugin" 
xl2tpd[1]: "pppol2tp.so" 
xl2tpd[1]: "pppol2tp" 
xl2tpd[1]: "7" 
xl2tpd[1]: "pppol2tp_lns_mode" 
xl2tpd[1]: "pppol2tp_tunnel_id" 
xl2tpd[1]: "47807" 
xl2tpd[1]: "pppol2tp_session_id" 
xl2tpd[1]: "58264" 
xl2tpd[1]: "passive" 
xl2tpd[1]: "nodetach" 
xl2tpd[1]: "192.168.42.1:192.168.42.10" 
xl2tpd[1]: "refuse-pap" 
xl2tpd[1]: "auth" 
xl2tpd[1]: "require-chap" 
xl2tpd[1]: "name" 
xl2tpd[1]: "l2tpd" 
xl2tpd[1]: "file" 
xl2tpd[1]: "/etc/ppp/options.xl2tpd" 
xl2tpd[1]: Call established with 172.17.0.1, Local: 58264, Remote: 1324, Serial: 1
/usr/sbin/pppd: In file /etc/ppp/options.xl2tpd: unrecognized option 'crtscts'
xl2tpd[1]: child_handler : pppd exited for call 1324 with code 2
xl2tpd[1]: call_close: Call 58264 to 172.17.0.1 disconnected
xl2tpd[1]: result_code_avp: result code endianness fix for buggy Apple client. network=768, le=3
xl2tpd[1]: control_finish: Connection closed to 172.17.0.1, serial 1 ()
xl2tpd[1]: Terminating pppd: sending TERM signal to pid 416
xl2tpd[1]: result_code_avp: result code endianness fix for buggy Apple client. network=256, le=1
xl2tpd[1]: control_finish: Connection closed to 172.17.0.1, port 49893 (), Local: 47807, Remote: 7
```

**Comment by jgimenez** [comment]
This was fixed in #21

**Comment by Justintime50** [comment]
Thank you! This worked out great. Actually came back to this repo as the one it was based off didn't work for separate reasons. Will give it another shot as I now have it working.
